client/wfnews-war/src/main/angular/package.json

```diff
@@ -47,9 +47,10 @@
     "@capacitor/app-launcher": "^5.0.2",
     "@capacitor/browser": "^5.0.2",
     "@capacitor/camera": "^5.0.2",
-    "@capacitor/core": "5.0.5",
+    "@capacitor/core": "^5.5.1",
```

Comment:
With the version bump here, do we need to adjust any of the other capacitor plugins? it's not a small bump!

Reply:
Updated all capacitor dependencies to the latest version